### ISSUE TITLE
"Always-on https" support

### ISSUE THREAD

**Issue by sajith** [snoyberg/keter#39]
It would be nice if Keter could support redirecting all http traffic to https.

(Relevant Yesod mailing list thread: https://groups.google.com/forum/#!topic/yesodweb/hzKTUNesPS0)

**Comment by chreekat** [comment]
@sajith, were you ever able to follow up on your problem about `keter: no ciphers available with those parameters.` ? I hit an identical issue, though I have not been able to isolate a test case yet.

**Comment by sajith** [comment]
Bryan Richter <notifications@github.com> wrote:

> @sajith, were you ever able to follow up on your problem about `keter: no ciphers available with those parameters.` ? I hit an identical issue, though I have not been able to isolate a test case yet.

Alas, no. :-(

All I found was that the behavior varies depending on environment, so
my guess is that it must have something to do with installed CA
certificates.  (I guess that error message may not mean what it says.)

I tested with Debian testing in my desktop, and Ubuntu 14.04 server in
a VM.  Keter in the Ubuntu VM always showed this message.  I tried the
same version of ca-certificates package in both setups, with the same
results.

I imagine that desktop might have extra certificates from Java or
browser or some such, but I'm not that good with computers... ;-)

I did not go as far as to create a standalone test case, so I never
figured out the root cause.  Sorry!
